bin/mongo-oplog-backup

```diff
@@ -66,4 +66,58 @@ opts = Slop.parse(help: true, strict: true) do
       puts "mongorestore [--drop] [--gzip] --oplogReplay #{File.join(dir, 'dump')}"
     end
   end
+
+  command 'restore' do
+  banner 'Usage: mongo-oplog-backup restore [options]'
+    on :v, :verbose, 'Enable verbose mode'
+    on :d, :dir, "Directory containing the backup to restore. Must contain a 'dump' folder.", argument: :required
+    on :full, 'Do a full restore, followed by an oplog restore.'
+    on :oplog, 'Restore only the oplogs. Assumes a manual restore has already been done.'
+
+    on :gzip, "Use gzip compression. This option is ignored by Oplog mode, because it autodetects."
+    on :ssl, "Connect to a mongod instance over an SSL connection"
+    on :sslAllowInvalidCertificates, "Allow connections to a mongod instance with an invalid certificate"
+    on :sslCAFile, "Specifies a Certificate Authority file for validating the SSL certificate provided by the mongod instance.", argument: :required
+    on :host, "Specifies a resolvable hostname for the mongod that you wish to backup.", default: 'localhost', argument: :required
+    on :port, "Specifies the port that mongod is running on", default: '27017', argument: :required
+    on :u, :username, "Specifies a username to authenticate to the MongoDB instance, if your database requires authentication. Use in conjunction with the --password option to supply a password.", argument: :required
+    on :p, :password, "Specifies a password to authenticate to the MongoDB instance. Use in conjunction with the --username option to supply a username. Note. the password will not be prompted for, so must be passed as an argument", argument: :required
+
+    on :noIndexRestore, "Don't restore indexes."
+    on :oplogLimit, "Only include oplog entries before the provided Timestamp: <seconds>[:ordinal]", argument: :required
```

Comment:
I believe that's referring to the argument for the option, not the option itself. https://github.com/leejarvis/slop/blob/v3.6.0/lib/slop/option.rb#L67